src/components/NewExperiment.vue

```diff
@@ -606,10 +606,6 @@
                                 proteomicsItem
                               )
                             ]"
-                            :forceSearchQuery="
-                              proteomicsItem.protein &&
-                                proteomicsItem.protein._pastedText
```

Comment:
(I guess this might get replaced with v-model (or reintroduced) for pasting mismatches)